lnwallet/channel_test.go

```diff
@@ -5851,6 +5851,41 @@ func TestChanReserveRemoteInitiator(t *testing.T) {
 		t.Fatalf("expected ErrBelowChanReserve, instead received: %v", err)
 	}
 
+	// Now we'll test the case where Alice is adding an HTLC which Bob
+	// considers dust, but not Alice.
+	aliceChannel, bobChannel, cleanUp, err = CreateTestChannels(
+		true,
+	)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer cleanUp()
+
+	// The amount of the HTLC should be above Alice's dust limit and below
+	// Bob's dust limit.
+	htlcSat := (btcutil.Amount(500) + htlcTimeoutFee(
```

Comment:
Where's this 500 coming from? Can we use an existing constant for p2wkh dust here instead?

Reply:
Added an explanation to the comment: https://github.com/lightningnetwork/lnd/pull/3691/commits/76455942017c29c03d8f3b1f2071643c93e1a39f